Apps/Src/Temperature.c

```diff
@@ -22,6 +23,14 @@ static uint8_t ChargingState;
 // Temperature.c uses auxiliary registers to view ADC data and COM register for I2C with LTC1380 MUX
 static cell_asic *Minions;
 
+void Temperature_delay_m(uint16_t milli)
+{
+  uint32_t delay = BSP_PLL_GetSystemClock() / 1000;
+	for(uint32_t i = 0; i < milli; i++)
+	{
+		for(uint32_t j = 0; j < delay; j++);
+	}
+}
```

Comment:
I think I'm missing this somewhere but where do we use this? I noticed the Temperature_delay_m hasn't been added to the header file and none of the functions in Temperature.c seem to be using it

Reply:
I don't think we're using this either. We added it in while debugging, and I never got around to removing it. I'll get rid of it